src/main/scala/com/chaordicsystems/cactus/Operator.scala

```diff
@@ -0,0 +1,17 @@
+package com.chaordicsystems.cactus
+
+object Operator extends Enumeration {
+  type Operator = Value
+  val AND, OR, LT, GT, EQ, NE, LE, GE, ALL, ANY = Value
+
+  val Unary: List[Operator] = List(LT, GT, EQ, NE, LE, GE)
```

Comment:
Actually, I was thinking about listing the operators in the exception, but then I gave up on this and forgot to remove. Removing it.